### PR TITLE
Format project to adhere to PEP8 standards and best practices

### PR DESCRIPTION
**Pull request by krittick**
## Summary

Update of #768 and #794 to include newer changes from `master` that couldn't easily be merged otherwise.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

**Comment by krittick** [comment]
This shouldn't contain any breaking changes, and has now been fairly well tested. All changes are simply to update the code to PEP8 standards, refactor and simplify conditionals, etc. 

I recommend that we merge this as soon as possible so that PRs can target a higher quality codebase and meet these standards.

File: discord/__main__.py [hunk 1]
```diff
@@ -152,44 +160,68 @@ async def cog_after_invoke(self, ctx):
...
 
+
 def to_path(parser, name, *, replace_spaces=False) -> Path:
     if isinstance(name, Path):
         return name
 
-    if sys.platform == 'win32':
-        forbidden = ('CON', 'PRN', 'AUX', 'NUL', 'COM1', 'COM2', 'COM3', 'COM4', 'COM5', 'COM6', 'COM7', \
-                     'COM8', 'COM9', 'LPT1', 'LPT2', 'LPT3', 'LPT4', 'LPT5', 'LPT6', 'LPT7', 'LPT8', 'LPT9')
+    if sys.platform == "win32":
```
Comment (UP929312) [inline discord/__main__.py:191]: This seems like a step back to be honest (I know it's auto-formatted, but then we'd need to add ignore/skip everywhere)

File: discord/__main__.py [hunk 2]
```diff
@@ -198,106 +230,147 @@ def newbot(parser, args) -> None:
...
+        "newbot", help="creates a command bot project quickly"
+    )
     parser.set_defaults(func=newbot)
 
-    parser.add_argument('name', help='the bot project name')
-    parser.add_argument('directory', help='the directory to place it in (default: .)', nargs='?', default=Path.cwd())
-    parser.add_argument('--prefix', help='the bot prefix (default: $)', default='$', metavar='<prefix>')
-    parser.add_argument('--sharded', help='whether to use AutoShardedBot', action='store_true')
-    parser.add_argument('--no-git', help='do not create a .gitignore file', action='store_true', dest='no_git')
+    parser.add_argument("name", help="the bot project name")
```
Comment (UP929312) [inline discord/__main__.py:307]: This also seems worse reading wise

**Review by UP929312 — COMMENTED** [review]
A few notes

File: discord/__main__.py (hunk 2 above)
Comment (UP929312) [reply discord/__main__.py:307]: Overall, these are 2 tiny things, but highlight a bigger issue, they make the code worse to read, and thus need to be handled with ignore/skip statements, so we can either add them, which will take a while, or let them be worse reading wise

File: discord/__main__.py (hunk 1 above)
Comment (krittick) [reply discord/__main__.py:191]: I disagree. These are both formatting changes only, and the current code violates the recommended line length limits. Reading the current code with less horizontal screen space is much more difficult before the changes.

No code logic has been changed at all with these two changes. It's simply been changed to adhere to the Black code style: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

**Review by ImNimboss — DISMISSED** [review]

**Comment by ImNimboss** [comment]
Relevant:

Maybe set up a brief period of time where no contributions are allowed for this PR to be merged - EmreTech, Pycord Discord server

**Review by KosmicAnomaly — COMMENTED** [review]
This is the sexiest pr I have seen come out of this repo

**Pull request merged**